### PR TITLE
Frontend/design

### PR DESCRIPTION
**Pull request by jarencudilla**
#### Frontend/design

Home page styling [WIP]

## The Story
I started styling our app using Bootstrap

- footer -- initial styling for our frontpage footer 
- team card -- initial styling for our frontpage team
- jumbotron - will check as jumbotron doesn’t work on Bootstrap 5 (probably under a different name)

TO DO --
-carousel
-FAQ
-Call to Action

## Screenshots

screenshot 116
screenshot 117
screenshot 115

## References

Color Palette inspiration :
https://visme.co/blog/website-color-schemes/  especially #6
https://www.awwwards.com/websites/flat-design/
Site inspiration:
https://cervooo.com/
https://pops.ph/

### CODE REVIEW

**Comment by rommeltorquator** [comment]
I like the design! It's very clean. Keep it up!

**Review by rommeltorquator — APPROVED** [review]

File: app/assets/stylesheets/custom.css.scss [hunk 1]
```diff
@@ -1 +1,42 @@
...
+.text-green{
+  color:#5CDB95;
+}
+.team-container {
+  background: #e8cbc0;
+  background: -webkit-linear-gradient(to right, #e8cbc0, #636fa4);
+  background: linear-gradient(to right, #e8cbc0, #636fa4);
+  align-items: center;
+  justify-content: center;
+}
```
Comment (analyn-cajocson) [inline app/assets/stylesheets/custom.css.scss]: add new line here

File: app/views/partials/_footer.html.erb [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+  <!-- Grid container -->
+
+  <!-- Copyright -->
+  <div class="text-center p-3" style="background-color: rgba(0, 0, 0, 0.2);">
+    © 2020 Copyright:
+    <a class="text-white" href="https://jaren.cudilla.net/">Jaren Cudilla</a>
+  </div>
+  <!-- Copyright -->
+</footer>
+<!-- Footer -->
```
Comment (analyn-cajocson) [inline app/views/partials/_footer.html.erb]: add new line here

File: app/views/partials/_jumbotron.html.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<div class="jumbotron">
+  <h1 class="display-4">Hello, world!</h1>
+  <p class="lead">This is a simple hero unit, a simple jumbotron-style component for calling extra attention to featured content or information.</p>
+  <hr class="my-4">
+  <p>It uses utility classes for typography and spacing to space content out within the larger container.</p>
+  <p class="lead">
+    <a class="btn btn-primary btn-lg" href="#" role="button">Learn more</a>
+  </p>
+</div>
```
Comment (analyn-cajocson) [inline app/views/partials/_jumbotron.html.erb]: add new line here

File: app/views/partials/_team.html.erb [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+                    <li class="list-inline-item"><a href="https://twitter.com/#" class="social-link"><i class="fa fa-twitter"></i></a></li>
+                    <li class="list-inline-item"><a href="https://www.instagram.com/rommeltorquator/" class="social-link"><i class="fa fa-instagram"></i></a></li>
+                    <li class="list-inline-item"><a href="https://www.linkedin.com/in/rommel-torquator-4870a799/" class="social-link"><i class="fa fa-linkedin"></i></a></li>
+                    <li class="list-inline-item"><a href="https://www.github.com/rommeltorquator/" class="social-link"><i class="fa fa-github"></i></a></li>
+                </ul>
+            </div>
+        </div><!-- End -->
+    </div>
+</div>
+</section>
```
Comment (analyn-cajocson) [inline app/views/partials/_team.html.erb]: add new line here

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/custom.css.scss (hunk 1 above)
Comment (jarencudilla) [reply app/assets/stylesheets/custom.css.scss]: done.

File: app/views/partials/_footer.html.erb (hunk 2 above)
Comment (jarencudilla) [reply app/views/partials/_footer.html.erb]: Done added new line

File: app/views/partials/_team.html.erb (hunk 4 above)
Comment (jarencudilla) [reply app/views/partials/_team.html.erb]: Done added new line

File: app/views/partials/_jumbotron.html.erb (hunk 3 above)
Comment (jarencudilla) [reply app/views/partials/_jumbotron.html.erb]: added new line

**Review by analyn-cajocson — APPROVED** [review]

**Pull request merged**